virtcontainers/pkg/oci/utils.go

```diff
@@ -414,6 +414,15 @@ func addHypervisorConfigOverrides(ocispec specs.Spec, config *vc.SandboxConfig)
 		config.HypervisorConfig.DisableVhostNet = disableVhostNet
 	}
 
+	if value, ok := ocispec.Annotations[vcAnnotations.EnableVhostUserStore]; ok {
+		enableVhostUserStore, err := strconv.ParseBool(value)
+		if err != nil {
+			return fmt.Errorf("Error parsing annotation for enable_vhost_user_store: Please specify boolean value 'true|false'")
```

Comment:
I don't think we need the `'true|false'` part of this error.

Reply:
Yes, it is not necessary there